cloud: Add secure-connections-to-serverless-tier-clusters.md (#10909)
pingcap/docs: tidb-cloud/connect-to-tidb-cluster.md

    > **Note:**
    >
    > - When you connect to a Developer Tier cluster, you must include the prefix for your cluster in the user name and wrap the name with quotation marks. For more information, see [User name prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix).

>     > - Serverless Tier only support TLS connection, For more information, see [Connect Securely to Serverless Clusters](/tidbcloud/connect-securely-to-serverless-clusters.md.md).

Reviewer: ```suggestion
    > - Serverless Tier only support TLS connection, For more information, see [Connect Securely to Serverless Clusters](/tidb-cloud/connect-securely-to-serverless-clusters.md.md).
```

>     > - Serverless Tier only support TLS connection, For more information, see [Secure connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).

Reviewer 2: ```suggestion
    > - Serverless Tier clusters only support TLS connection. For more information, see [Secure connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).
```

>     > - Serverless Tier only support TLS connection, For more information, see [Secure connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).

Author: ```suggestion
    > - Serverless Tier only support TLS connection, For more information, see [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).
```

>     > - Serverless Tier only support TLS connection, For more information, see [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).

Author: ```suggestion
    > - Serverless Tier clusters only support TLS connection. For more information, see [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).
```
